pkgs/top-level/config.nix

```diff
@@ -61,6 +61,7 @@ let
 
     contentAddressedByDefault = mkMassRebuild {
       feature = "set `__contentAddressed` to true by default";
+      #default = true;
```

Comment:
Committed by accident?

Reply:
How did I manage to do that? Yeah, accident.